openshift/tests-extension/README.md

```diff
@@ -15,6 +15,121 @@ They use the framework: https://github.com/openshift-eng/openshift-tests-extensi
+
+**Prerequisites:**
+
+Install OLMv1 before running the tests. 
+You can use the `kind` tool to create a local Kubernetes cluster with OLMv1 installed.
+Furthermore, if you are using feature gates in your test you will need to
+ensure that those are enabled in your cluster.
+
+**Example:**
+
+export KUBECONFIG=$HOME/.kube/config
+
+```shell
+$ ./bin/olmv1-tests-ext run-suite olmv1/all
+  Running Suite:  - /Users/camilam/go/src/github/operator-framework-operator-controller/openshift/tests-extension
+  ===============================================================================================================
+  Random Seed: 1753508546 - will randomize all specs
+
+  Will run 1 of 1 specs
+  ------------------------------
+  [sig-olmv1][OCPFeatureGate:NewOLM] OLMv1 CRDs should verify required CRDs are installed and active
+  /Users/camilam/go/src/github/operator-framework-operator-controller/openshift/tests-extension/test/olmv1.go:37
+  [INFO] [env] Using kubeconfig: /Users/camilam/.kube/config[WARN] Skipping feature capability check: not OpenShift  STEP: verifying CRD clusterextensions.olm.operatorframework.io @ 07/26/25 06:42:26.57
+    STEP: verifying CRD clustercatalogs.olm.operatorframework.io @ 07/26/25 06:42:26.575
+  • [0.026 seconds]
+  ------------------------------
+
+  Ran 1 of 1 Specs in 0.026 seconds
+  SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 0 Skipped
+  Running Suite:  - /Users/camilam/go/src/github/operator-framework-operator-controller/openshift/tests-extension
+  ===============================================================================================================
+  Random Seed: 1753508546 - will randomize all specs
+
+  Will run 1 of 1 specs
+  ------------------------------
+  [sig-olmv1] OLMv1 should pass a trivial sanity check
+  /Users/camilam/go/src/github/operator-framework-operator-controller/openshift/tests-extension/test/olmv1.go:26
+  • [0.000 seconds]
+  ------------------------------
+
+  Ran 1 of 1 Specs in 0.000 seconds
+  SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 0 Skipped
+[
+  {
+    "name": "[sig-olmv1][OCPFeatureGate:NewOLM] OLMv1 CRDs should verify required CRDs are installed and active",
+    "lifecycle": "blocking",
+    "duration": 26,
+    "startTime": "2025-07-26 05:42:26.553886 UTC",
+    "endTime": "2025-07-26 05:42:26.580057 UTC",
+    "result": "passed",
+    "output": "[INFO] [env] Using kubeconfig: /Users/camilam/.kube/config[WARN] Skipping feature capability check: not OpenShift  STEP: verifying CRD clusterextensions.olm.operatorframework.io @ 07/26/25 06:42:26.57\n  STEP: verifying CRD clustercatalogs.olm.operatorframework.io @ 07/26/25 06:42:26.575\n"
+  },
+  {
+    "name": "[sig-olmv1] OLMv1 should pass a trivial sanity check",
+    "lifecycle": "blocking",
+    "duration": 26,
+    "startTime": "2025-07-26 05:42:26.553852 UTC",
+    "endTime": "2025-07-26 05:42:26.580263 UTC",
+    "result": "passed",
+    "output": ""
+  }
+]
+```
+
```

Comment:
@kuiwang02 It’s okay to add it in a follow-up, but I’d like to document a comprehensive guide here on how to run the tests locally using openshift-bot and the payload.

Ideally something like:

Step 1

Step 2

Step 3
...

If you’re able to provide more details, I’d really appreciate it!
I’ll try it locally myself and then submit this guide in a follow-up PR.

Thanks!